### PR TITLE
Add blog post for the OGC tests in CI grant app

### PR DESCRIPTION
**Pull request by pblottiere**

### CODE REVIEW

File: content/blog/ciwms130.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: "QGIS Grant Program: QGIS Server, OGC tests and continuous integration"
```
Comment (SrNetoChan) [inline content/blog/ciwms130.md]: We could call it QGIS Grant Program - Part 1 - QGIS Server ..

File: content/blog/ciwms130.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+
+# Introduction
+
+Every year, [QGIS.org](https://qgis.org/en/site/) provides funding for
+improving the QGIS ecosystem on too often neglected subjects like refactoring,
+packaging or upstream studies. In light of this event, called QGIS Grant
+Program, some ideas proposed by the
+[QCooperative](https://www.qcooperative.net/) team have been accepted by the
+voting members. One of them was: *QGIS Server, OGC tests and Continuous
```
Comment (SrNetoChan) [inline content/blog/ciwms130.md]: Here we can list the three accepted proposals. Later we can add links to their articles.

**Comment by pblottiere** [comment]
Hello @SrNetoChan. I added some context for future articles: https://github.com/qcooperative/website/pull/9/commits/51f2e51ac432ac2650f143eb1bf7a9d772201dfd

Let me know what you think.

Thanks :)

**Review by SrNetoChan — APPROVED** [review]
Looks good to me. Lets publish it- It seems I should start working on the QA one myself.

**Pull request merged**